src/infinity-menu.js

```diff
@@ -261,6 +261,7 @@ export default class InfinityMenu extends React.Component {
 
 		/*header component*/
 		const headerProps = {
+			headerMetadata: this.props.headerMetadata,
```

Comment:
i like explicitly namespacing the custom metadata (by referencing properties on `headerMetadata`) just because there are already additional props being passed into the header component (isSearching, searchInput, etc...), and this allows metadata to stand out more clearly relative to those props. 

either approach is fine as long as it is clearly documented in the README which props are being passed into the HeaderComponent